content/fast/use-lazysizes-to-lazyload-images/codelab-use-lazysizes-to-lazyload-images.md

```diff
@@ -101,9 +101,12 @@ add lazy loading to any offscreen images.
 
 <web-screenshot type="show-live"></web-screenshot>
 
-- Run the Lighthouse performance audit (Lighthouse > Options > Performance) on
-  the live version of your Glitch and verify that the "Defer offscreen images"
-  audit was passed.
+With the live site open:
+1. Open Chrome DevTools.
```

Comment:
```suggestion
1. Open Chrome DevTools by pressing `CMD + OPTION + i` / `CTRL + SHIFT + i`.
```

Reply:
I put together a little blurb of text in a wiki to try and get us using the same language in our guides.
https://github.com/GoogleChrome/web.dev/wiki/How-to-write-a-guide